imageroot/actions/restore-module/21database

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env sh
+
+#
+# Copyright (C) 2023 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+set -e
+
+# the removal of the container after the action is to ensure that if anything goes wrong
+# you can access the container failed to check if anything can be done manually
+
+# prepare restore
+podman run \
+    --name=prepare_db \
+    --volume=./db_backup:/backup:z \
+    "${NETHVOICE_MARIADB_IMAGE}" \
+    mariabackup --prepare --target-dir=/backup
+podman rm prepare_db
```

Comment:
Why not using directly the `--rm` option?
```suggestion
podman run \
    --name=prepare_db \
    --rm \
    --volume=./db_backup:/backup:z \
    "${NETHVOICE_MARIADB_IMAGE}" \
    mariabackup --prepare --target-dir=/backup
```

Reply:
The `--rm` was kept out because if the backup command fails a manual intervention is possible to declare what went wrong inside the container. Otherwise you'll need to manually spin it up to figure out what went wrong. I've left a command above to explain the behaviour.

Not critical, will apply the suggestion.

Reply:
Thanks, I get it, but I'd prefer to leave the system in a cleaner state.